.changeset/dry-rules-bathe.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/netlify': patch
+---
+
+Fixes cacheOnDemandPages setting to only cache GET/HEAD requests by default
```

Comment:
```suggestion
Updates the behavior of the `cacheOnDemandPages` setting to only cache GET/HEAD requests by default
```
Maybe a slight preference for "updates the behaviour" as presenting it more of a "thing you might want to check if you have this enabled because it changed" vs just saying "fix" (which more implies something that was broken before should now work, not "we changed how it worked").

Just a thought! Up to you! Either way, I would use code syntax for the property!